R/grid.R

```diff
@@ -0,0 +1,295 @@
+#' Generate CSS string representing grid template areas.
+#' @param areas_dataframe data.frame of strings representing grid areas
+#' @return string
```

Comment:
I think that text format is called `character` in R

Reply:
I noticed there's more of these, could you normalise that across the script?

Reply:
done :heavy_check_mark: